plugins/node/opentelemetry-instrumentation-ioredis/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@opentelemetry/instrumentation-ioredis",
+  "version": "0.12.0",
+  "description": "OpenTelemetry ioredis automatic instrumentation package.",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js",
+  "scripts": {
+    "test": "nyc ts-mocha -p tsconfig.json 'test/**/*.test.ts'",
+    "test:debug": "cross-env RUN_REDIS_TESTS_LOCAL=true ts-mocha --inspect-brk --no-timeouts -p tsconfig.json 'test/**/*.test.ts'",
+    "test:local": "cross-env RUN_REDIS_TESTS_LOCAL=true npm run test",
+    "tdd": "npm run test -- --watch-extensions ts --watch",
+    "clean": "rimraf build/*",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix",
+    "precompile": "tsc --version",
+    "version:update": "node ../../../scripts/version-update.js",
+    "compile": "npm run version:update && tsc -p .",
+    "codecov": "nyc report --reporter=json && codecov -f coverage/*.json -p ../../",
+    "prepare": "npm run compile"
+  },
+  "keywords": [
+    "opentelemetry",
+    "ioredis",
+    "redis",
+    "nodejs",
+    "tracing",
+    "profiling",
+    "instrumentation"
+  ],
+  "author": "OpenTelemetry Authors",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "files": [
+    "build/src/**/*.js",
+    "build/src/**/*.d.ts",
+    "doc",
+    "LICENSE",
+    "README.md"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "devDependencies": {
+    "@opentelemetry/context-async-hooks": "^0.14.0",
+    "@opentelemetry/node": "^0.14.0",
+    "@opentelemetry/semantic-conventions": "^0.14.0",
```

Comment:
`semantic-conventions` is a runtime deps, see https://github.com/open-telemetry/opentelemetry-js-contrib/pull/310